packages/alfa-rules/src/sia-r111/rule.ts

```diff
@@ -45,12 +46,36 @@ export default Rule.Atomic.of<Page, Element>({
  * @public
  */
 export namespace Outcomes {
```

Comment:
**Note:** Since the outcomes are shared, they can be refactored in `src/common/outcomes`.
Not urgent.

Reply:
I'll add it to this #1593